app/public/cantusdata/management/commands/import_data.py

```diff
@@ -125,3 +130,16 @@ def import_chant_data(self, **options):
                 chant_count
             )
         )
+
+    @transaction.atomic
+    def import_iiif_data(self):
+        with open(path.join(BASE_DIR, "data_dumps", "manifests.csv"), "r") as file:
+            csv = file.readlines()
+
+        for row in csv:
+            siglum_manifest = row.split(",")
+            qs = Manuscript.objects.filter(siglum=siglum_manifest[0].strip())
+            if len(qs) > 0:
```

Comment:
So this is catching the times that the query in line 141 doesn't yield anything (ie when `len(qs) = 0`)?

Reply:
yeah exactly